src/layouts/Titled/components/TOC.tsx

```diff
@@ -121,6 +121,9 @@ export const TOC: React.FC<{ data: IHeading[] }> = ({ data }) => {
         position: sticky;
         top: ${p => p.theme.space.lg};
         margin-left: ${p => p.theme.space.base * 15}px;
+        max-height: calc(100vh - 64px);
+        overflow-y: scroll;
+        -ms-overflow-style: scrollbar;
```

Comment:
Please follow the advice provided in the description of https://garden.zendesk.com/css-components/scrollbars/ and use `-ms-overflow-style: -ms-autohiding-scrollbar` rather than `scrollbar`. Instead of adding here, apply this to the [`GlobalStyling`](https://github.com/zendeskgarden/website/blob/master/src/layouts/Root/index.tsx) as:

```css
* {
  -ms-overflow-style: -ms-autohiding-scrollbar;
}
```

Reply:
@hzhu looks like you forgot to remove this line here.